emperor/support_files/js/color-view-controller.js

```diff
@@ -56,62 +55,108 @@ define([
     var SLICK_WIDTH = 25, scope = this;
     var name, value, colorItem;
 
+    // Create scale div and checkbox for whether using scalar data or not
+    this.$scaleDiv = $('<div></div>')
+    this.$colorScale = $("<svg width='90%' height='40'></svg>");
+    this.$scaleDiv.append(this.$colorScale);
+    this.$scaled = $("<input type='checkbox' id='scaled'>");
```

Comment:
Yeah, you are right Josh, for these properties, there's no need to have
unique identifiers. However, we have these for the
EmperorViewControllers in general, we have an `identifier` property and
it follows a scheme similar to what Daniel suggests.

On (Apr-18-16| 9:59), Joshua Shorenstein wrote:
> > @@ -56,62 +55,108 @@ define([
> >      var SLICK_WIDTH = 25, scope = this;
> >      var name, value, colorItem;
> >  
> > +    // Create scale div and checkbox for whether using scalar data or not
> > +    this.$scaleDiv = $('<div></div>')
> > +    this.$colorScale = $("<svg width='90%' height='40'></svg>");
> > +    this.$scaleDiv.append(this.$colorScale);
> > +    this.$scaled = $("<input type='checkbox' id='scaled'>");
> 
> As far as I can tell in the codebase (and @ElDeveloper correct me if I'm wrong) nothing is done through IDs directly, but instead using the objects attached to the controllers, so this is probably not needed.
> 
> ---
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub:
> https://github.com/biocore/emperor/pull/456/files/c8896fbb1b35090514974c949912f38de068eb3b#r60094408